vertx-exercises/pom.xml

```diff
@@ -21,4 +21,25 @@
       </properties>
     </profile>
   </profiles>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
```

Comment:
You may want to use the Vert.x Maven Plugin instead of the exec plugin.

Reply:
thanks for the review. will take a look into it.